test/fp64-test/webpack.config.test-fp64.js

```diff
@@ -0,0 +1,52 @@
+// This is a Webpack 2 configuration file
+const {resolve} = require('path');
+// const webpack = require('webpack');
+
+module.exports = {
+  devServer: {
+    stats: {
+      warnings: false
+    },
+    quiet: true
+  },
+
+  // Bundle the tests for running in the browser
+  entry: {
+    'test-fp64': resolve('fp64-shader.spec.js')
+  },
+
+  // Generate a bundle in dist folder
+  output: {
+    path: resolve('./dist'),
+    filename: '[name]-bundle.js'
+  },
+
+  devtool: '#inline-source-maps',
+
+  resolve: {
+    alias: {
+      'deck.gl': resolve('../../src')
+    }
+  },
+
+  module: {
+    rules: [
+      {
+        // Mapbox has some unresolved fs calls
```

Comment:
This comment is actually wrong - it is not needed for mapbox here.

It could have taken care of the readFileSync inlining, if you had added current dir to the `include`s.

A problem with babel-brfs is that we don't seem to get hot reloading of shaders, perhaps your import solution is better there.